src/libs/ReportActionComposeFocusManager.ts

```diff
@@ -83,6 +83,10 @@ function isEditFocused(): boolean {
     return !!editComposerRef.current?.isFocused();
 }
 
+function preventFocusOnFirstResponderOnce() {
```

Comment:
NAB - it would be good to add a comment explaining what this is for and that it's mostly needed for iOS. We also used the term "first responder" internally for something completely different, so it would help avoid any ambiguity.